### ISSUE TITLE
When text-only zooming, the sticky part does not wrap nicely

### ISSUE THREAD

**Issue by yatil** [w3c/wai-wcag-quickref#182]
This looks like a relatively simple to fix flexbox bug. (via Janis @ a11y slack)

![](https://user-images.githubusercontent.com/58862/53473910-fbaf6b80-3a6b-11e9-9e01-f33e48d278ab.png)
